streams/test-utils/src/test/java/org/apache/kafka/streams/TopologyTestDriverTest.java

```diff
@@ -1878,4 +1877,33 @@ public void shouldRespectTaskIdling() {
             );
         }
     }
+
+    @Test
+    public void shouldMatchSystemTime() {
```

Comment:
I think we can remove those two new tests.

If you really want to test it, you should rather add a `Processor` to the Topology that gets the current system and stream time and verify it this way, ie, we should rather add an integration test than a unit test.